docs/run/nodes.md

```diff
@@ -1,48 +1,45 @@
-# Running a node
-This guide describes how to run a hosted Ceramic node that can be used by a Ceramic client to interact with the Ceramic network. Currently, it is only possible to run nodes on the Clay testnet. We will add the ability to run nodes on mainnet in the near future.
+# Hosting a node
+This guide describes how to spin up and run a hosted Ceramic node in JavaScript that can be used as a remote node by the [JS HTTP Client](../clients/javascript/http.md) or the [CLI](../clients/javascript/cli.md). It can also be used for redundant stream pinning and replication from any other node.

```

Comment:
I don't love the last sentence here, but am struggling to find a better way to word it. Maybe something like:
"It can also be used to replicate and pin streams from any other node to create data redundancy and high availability"

Reply:
same comment applies to any other place in this PR that uses the phrase "redundant stream pinning and replication", which I don't feel like is very clear

Reply:
fixed